### PR TITLE
Support Celo Bridge Assets

### PR DESCRIPTION
**Pull request by Jesse-Sawa**
Will create follow up PR to add further address mappings

### CODE REVIEW

File: src/constants/tokens.ts [hunk 1]
```diff
@@ -117,3 +117,11 @@ export const USDT_BNB = new Token(
   'USDT',
   'Tether USD'
 )
+
+export const USDT_CELO = new Token(
+  ChainId.CELO,
+  '0x617f3112bf5397D0467D315cC709EF968D9ba546',
+  6,
+  'USDT',
+  'Teather USD'
```
Comment (nicolasbrugneaux) [inline src/constants/tokens.ts:126]:
```suggestion
  'Tether USD'
```

File: src/index.test.ts [hunk 2]
```diff
@@ -130,6 +130,34 @@ it('outputs bnb list correctly', async () => {
...
+    t.address,
+    t.chainId,
+    t.extensions,
+  ])
+  const sampleMap = tokenList?.tokens.map((t) => [t.address, t.chainId, t.extensions])
+  console.log(sampleMap)
+  console.log(sample)
+  expect(tokenList).toBeDefined()
+  expect(tokenList?.version).toEqual(celoedSampleTokenList.version)
+  console.log(tokenList )
```
Comment (nicolasbrugneaux) [inline src/index.test.ts:148]:
```suggestion
  expect(tokenList).toBeDefined()
  expect(tokenList?.version).toEqual(celoedSampleTokenList.version)
```

removed console logs

File: src/providers/CeloMappingProvider.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
+import { MappingProvider } from './MappingProvider'
+import { MappedTokenData } from '../constants/types'
+import celoMappings from '../local_mappings/celo.json'
+
+// barring a better external source, we use a local source of mappings for avalanche
```
Comment (nicolasbrugneaux) [inline src/providers/CeloMappingProvider.ts:5]: comment seems unrelated
```suggestion
// barring a better external source, we use a local source of mappings for celo
```

**Review by nicolasbrugneaux — CHANGES_REQUESTED** [review]

File: src/fixtures/index.ts [hunk 4]
```diff
@@ -236,6 +239,34 @@ export const bnbedSampleTokenList_2 = {
   ].sort(compareTokenInfos),
 }
 
+//todo check if this is needed
```
Comment (just-toby) [inline src/fixtures/index.ts:242]: please remove this comment

File: src/index.test.ts [hunk 5]
```diff
@@ -28,7 +28,7 @@ import {
   avalanchedSampleTokenList,
   sampleL1TokenList_3,
   baseGoerliSampleTokenList_3,
-  baseSampleTokenList_3,
+  baseSampleTokenList_3, celoedSampleTokenList,
```
Comment (just-toby) [inline src/index.test.ts:31]: can you fix the formatting here?

File: src/fixtures/index.ts (hunk 4 above)
Comment (JFrankfurt) [inline src/fixtures/index.ts:242]:
```suggestion

```

**Review by just-toby — APPROVED** [review]
please fix the formatting and comment

File: src/index.test.ts (hunk 5 above)
Comment (Jesse-Sawa) [reply src/index.test.ts]: Done!

**Review by JFrankfurt — APPROVED** [review]

**Pull request merged**